src/Microsoft.HttpRepl/OpenApi/ApiDefinitionReader.cs

```diff
@@ -3,25 +3,22 @@
 
 using System;
 using System.Collections.Generic;
-using Newtonsoft.Json.Linq;
 
 namespace Microsoft.HttpRepl.OpenApi
 {
-    public class ApiDefinitionReader
+    internal class ApiDefinitionReader
```

Comment:
So we expect no consumers of Http Repl to depend on this class (and other public classes in this repo?)

Reply:
See comment above